### ISSUE TITLE
ECR CRR ReplicationConfiguration.ReplicationRule.RepositoryFilters marked as required incorrectly

### ISSUE THREAD

**Issue by TsimpDim** [cloudtools/troposphere#1993]
The specific property is optional, as proven by the spec

```
    "AWS::ECR::ReplicationConfiguration.ReplicationRule": {
      "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecr-replicationconfiguration-replicationrule.html",
      "Properties": {
        "RepositoryFilters": {
          "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecr-replicationconfiguration-replicationrule.html#cfn-ecr-replicationconfiguration-replicationrule-repositoryfilters",
          "UpdateType": "Mutable",
          "Required": false,
          "Type": "List",
          "ItemType": "RepositoryFilter"
        },
        "Destinations": {
          "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecr-replicationconfiguration-replicationrule.html#cfn-ecr-replicationconfiguration-replicationrule-destinations",
          "UpdateType": "Mutable",
          "Required": true,
          "Type": "List",
          "ItemType": "ReplicationDestination"
        }
      }
    }
```

and [the CF page itself](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecr-replicationconfiguration-replicationrule.html)

The relevant code lines are here: https://github.com/cloudtools/troposphere/blob/f286aed5a9e09d1b2337fceec51d169bf2faddfe/troposphere/ecr.py#L47-L51

I'm free to help, although I'm not sure if pushing a PR that changes only that line (L50) would be the right thing to do.

**Comment by markpeek** [comment]
Thanks for the issue. Fixed via c1590b5a288e34792aaae1c0b33e6814f8592053

**Comment by TsimpDim** [comment]
Thanks!
